shared/javascripts/local_storage.js

```diff
@@ -26,7 +26,7 @@ var ls = {
     if ( ls.localStorageDefined ) {
       localStorage.setItem(key, value);
     } else {
-      ls.preferences.setCharPref(key, value);
+      ls.storage[key] = value;
```

Comment:
Does ls have the `setItem` function as well? I'd like to more closely mirror the two if possible